src/stores/UserStore.js

```diff
@@ -2,10 +2,9 @@ import { defineStore } from 'pinia';
 
 export const useUserStore = defineStore('user', {
   state: () => ({
-    id: '',
     username: '',
     firstname: '',
```

Comment:
Shouldn't we also replace `username` by `name` and `firstname` by `login`? Since this is what we get from the backend server

Reply:
done